frontend/src/views/Collection/components/ActionButtons/components/DeleteButton/index.tsx

```diff
@@ -29,7 +31,7 @@ const DeleteCollectionButton = ({
 
   return (
     <>
-      <Button
+      <StyledOutlineButton
```

Comment:
Modified the style for "Delete Collection" button to match the adjacent buttons styles.